Automattic/woocommerce-payments
# Display express payment buttons on checkout blocks pay-for-order page

#### Changes proposed in this Pull Request
When the page has blocks we don't initialize the express payment buttons, so if it's a pay-for-order page but the checkout page uses blocks, we won't see the express payment buttons even though pay-for-order never shows the checkout blocks.  This PR adds the logic to display the buttons on pay-for-order pages when the checkout page is using checkout blocks.
 
[Design discussion in this PR](https://github.com/Automattic/woocommerce-payments/pull/7535).

<img width="826" alt="Screen Shot 2023-10-27 at 4 10 21 PM" src="https://github.com/Automattic/woocommerce-payments/assets/56378160/897de22e-52e1-417e-bb6f-20a468a3b744">


#### Testing instructions
1. Make sure your merchant is online so the express payment buttons (ex. Google pay button) would show
2. Use blocks checkout on your checkout page that is set under WooCommerce settings -> Advanced
3. Create a pay-for-order order
4. Go to the pay-for-order page and see the express payment buttons

*

-------------------


**Post merge**
